src/amber/cli/templates/scaffold/controller/crecto/spec/controllers/{{name}}_controller_spec.cr.ecr

```diff
@@ -1,5 +1,6 @@
 <% field_name = @fields.first.name -%>
 <% human_name = @name.split("_").map(&.capitalize).join(" ") %>
+<% create_method_name = "create_#{@name.downcase}" %>
```

Comment:
This likely does no harm, but the name should already be `underscore`d

In general for the templates - for the names, we need to define all of them as instance methods in the class itself so it is consistent.

The full convention for naming will probably continue to evolve - subing out all characters that aren't alphanumeric or underscore, etc.

Then there is the whole pluralization, but I digress - for a different day.